Pipfile

```diff
@@ -5,6 +5,14 @@ name = "pypi"
 
 [packages]
 requests = "*"
```

Comment:
Commenting here about files that should not exist in this PR and all major changes required.

- Remove `asserts_copy` folder and any other frontend project related files from this pull request. 
- All frontend files should be held in your frontend project, not here.
- Include `*.sqlite3` in `.gitignore` file and make sure no database files are published in this PR.
- You need to clean up the whole project, a lot of unnecessary files for no reason. Just keep them locally on your laptop somewhere. Don't commit all your "try out" files. 
- Pull requests should contain only the final version files.
- `manage.py` file outside of Django project folder? 
- `.DS_Store` should not be committed, add to `.gitignore` file and make sure such files are not in this PR.

Reply:
Took care of all points. Except for statement 'manage.py file outside of Django project folder?' 
This is the structure I have been working with all along.
   whatsagent
         whatsagent
                   api
                              migrations
                   views.py
         manage.py
         whatsagent.env